emmaa/analyze_tests_results.py

```diff
@@ -352,18 +353,21 @@ def make_changes_over_time(self):
 
     def get_over_time(self, section, metrics):
         if not self.previous_json_stats:
-            return []
-        previous_data = self.previous_json_stats['changes_over_time'][metrics]
-        updated = previous_data.append(self.json_stats[section][metrics])
-        return updated
+            previous_data = []
+        else:
+            previous_data = (
+                self.previous_json_stats['changes_over_time'][metrics])
+        previous_data.append(self.json_stats[section][metrics])
+        return previous_data
 
     def save_to_s3(self):
+        json_stats_str = json.dumps(self.json_stats)
```

Comment:
One comment: if this JSON is supposed to be easily human-readable, you might want to add `indent=1` as an argument to `json.dumps`

Reply:
Added